tox.ini

```diff
@@ -262,6 +263,7 @@ deps =
     external_boto3-boto01: moto<2.0
     external_boto3-py27: rsa<4.7.1
     external_botocore-botocorelatest: botocore
+    external_botocore-botocore129: botocore<1.29
```

Comment:
We can call this one  `external_botocore-botocore128` since it's for `botocore<1.29`.

Reply:
Oops. Good catch!